documentation/book/proc-creating-new-image-from-base.adoc

```diff
@@ -3,14 +3,20 @@
 // assembly-using-kafka-connect-with-plugins.adoc
 
 [id='creating-new-image-from-base-{context}']
-= Create a new image based on our base image
+= Creating a Docker image from the Kafka Connect base image
 
-{ProductName} provides its own container image for running Kafka Connect, which can be found on {DockerRepository} as
-`{DockerKafkaConnect}`.
-The `{DockerKafkaConnect}` image is configured to automatically load all plugins or connectors that are present in the `/opt/kafka/plugins` directory during startup.
-This image can be used as a base image for building a new custom image with additional plugins.
+A container image for running Kafka Connect using {ProductName} is available on {DockerRepository} as `{DockerKafkaConnect}`. You can use this as a base image for creating your own custom image with additional plug-ins. 
 
-The following procedure describes the process for creating such a custom image and adding the connector plugins to the the `/opt/kafka/plugins` directory.
+At startup, Kafka Connect loads any third-party plug-ins, including connectors, contained in the `/opt/kafka/plugins` directory. This procedure describes how to create a custom image and add your connector to this directory. 
```

Comment:
_including connectors_ sounds a bit strange. Basically, the plugins are connectors.

Reply:
Agreed. I've simplified the terminology to just say "connector plug-ins" as you suggested earlier.